### ISSUE TITLE
Interest in adding `master` as a default fallback branch?

### ISSUE THREAD

**Issue by craigmulligan** [gajus/gitinfo#12]
.getRemoteUrl() fails if the local branch is not set to track a remote branch. 

Which means this fails unless it has an upstream branch
```
require('gitinfo')()
```

I *assume* there maybe other usecases like mine where branch tracking is irrelevant (I'm only after remote repo name and user)

Interesting in we adding something like ?
```
const gitInfo = require('gitinfo')({
  defaultBranch: 'master'
});
```
Or we could make it automatically fallback to `master`.

This way we could fallback to master if no branch is found in the config, get around the `'Branch ("' + branchName + '") definition does not exist in the config.'` error?

**Comment by gajus** [comment]
> const gitInfo = require('gitinfo')({
>   defaultBranch: 'master'
> });

Sounds reasonable. Just `defaultBranchName`.

When not configured, it should still throw an error, though.

**Comment by craigmulligan** [comment]
Great! I'll prep a PR.

**Comment by craigmulligan** [comment]
Closed with #15
